### ISSUE TITLE
Register new safe_html transform

### ISSUE THREAD

**Issue by tisto** [collective/experimental.safe_html_transform#3]

**Comment by tisto** [comment]
setuptools.py:

```
def installTransform(portal, transform_id):
    transforms = getToolByName(portal, 'portal_transforms')
    if transform_id not in transforms.objectIds():
        transforms.manage_addTransform(
            transform_id,
            experimental.safe_html_transform.%s' % transform_id
        )

def importVarious(context):
    ...
   installTransform(portal, 'safe_html')
```

**Comment by tisto** [comment]
Example how to test if a transform has been registered (testing unregister would be to use "assertFalse" instead).

```
    def test_safe_html_transform_registered(self):
        self.assertTrue(
            'safe_html' in self.transforms.objectIds())
```

**Comment by prakharjoshi** [comment]
We have to register the new browser layer only, right ?

**Comment by tisto** [comment]
I don't think you can register a transform for a specific browserlayer. We could revert the installation process with an uninstall profile though (we will take care of this later).

**Comment by prakharjoshi** [comment]
So right now, we have unregistered the old safe_html from portal Transform and now first we have to write the interface for the product then we can register that ?

**Comment by prakharjoshi** [comment]
@gforcada  @tisto What is the "transform()" function on line number 26 of https://github.com/plone/plone.outputfilters/blob/master/plone/outputfilters/setuphandlers.py

**Comment by prakharjoshi** [comment]
@lentinj  I guess it is covered in the issue #21 . As on that pull request we will add the transform and register it. So this issue can be merged with issue #21. Please have a look at this and again if you permit I can close that issue too :).
